docs/conf.py

```diff
@@ -0,0 +1,21 @@
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.duration',
+    'sphinx.ext.autosectionlabel',
+]
+
+html_theme = 'alabaster'
```

Comment:
i dont think we need to worry too much at this stage - apart from anything the Envoy theme is pretty awful atm, but we may want to sync themes at some point so the website has some consistency

Reply:
I thought about that and decided that I didn't want to muck with themes right now. 😂 I'll take a look at the Envoy theme shortly.